### ISSUE TITLE
Credits to add

### ISSUE THREAD

**Issue by alix-tz** [HTR-United/cremma-wikipedia#2]
```
authors:
  - name: 
    surname: Antoine
    type: person
  - name: Guichard
    surname: Morgane
    type: person
  - name: Chagué
    surname: Alix
    type: person
  - name: Chateau
    surname: Emmanuel
    type: person
  - name: Razafindrakoto
    surname: Elise
    type: person
  - name: Foisy-Marquis
    surname: Clémence
    type: person
  - name: Hennequart
    surname: Pauline
    type: person
  - name: Mellet
    surname: Margot
    type: person
  - name: Jia
    surname: Arilys
    type: person
  - name: Verstaete
    surname: Mathilde
    type: person
  - name: Krause
    surname: Lena
    type: person
  - name: Mozin
    surname: Zoé
    type: person
  - name: Bourgeois
    surname: Camille
    type: person
  - name: Delattre
    surname: Camille
    type: person
  - name: Ferretti
    surname: Giulia
    type: person
  - name: Vitali Rosati
    surname: Marcello
    type: person
  - name: Delannay
    surname: Roch
    type: person
  - name: Lescouet
    surname: Emmanuelle
    type: person
```

**Comment by PonteIneptique** [comment]
Whole HTR-United.yml needs to be redone: https://github.com/HTR-United/cremma-wikipedia/blob/master/htr-united.yml

**Comment by alix-tz** [comment]
More credits to add:
```txt
  - name: Yves
    surname: Tadjo
    type: person
  - name: Rachel
    surname: Bawden
    type: person
  - name: Benoit
    surname: Sagot
    type: person
  - name: Jules
    surname: Nuguet
    type: person
  - name: Camille
    surname: Rey
    type: person
  - name: Floriane
    surname: Chiffoleau
    type: person
  - name: Wissam
    surname: Antoun
    type: person
  - name: Lydia
    surname: Nishimwe
    type: person
  - name: Corinne
    surname: Jenna
    type: person
  - name: Roman
    surname: Castagné
    type: person
  - name: Kelly
    surname: Christensen
    type: person
  - name: Rian
    surname: Touchent
    type: person
  - name: Anna
    surname: Chepalkina
    type: person
  - name: Eric
    surname: de la Clergerie
    type: person
  - name: Hugo
    surname: Scheithauer
```

**Comment by alix-tz** [comment]
@PonteIneptique I followed the structure of the schema to document the authors of a dataset, but 1) there's no role for a contribution such as "scripture providers" and 2) I actually don't think they should appear as authors but actually more in an acknowledgement section. What do you think?

**Comment by PonteIneptique** [comment]
I'd actually use support, as it is meant for generic purpose but we could have "writers" or something like this.

**Comment by alix-tz** [comment]
Fixed with #22 -> I added every "writer" as a support
